README.org

```diff
@@ -1275,7 +1275,7 @@ manager must be used.
 
 ;; Async http clients may also be created and re-used:
 (let [acm (conn/make-reuseable-async-conn-manager {})
-      ahclient (core/build-async-http-client {} acm "https://example.com" false)]
+      ahclient (core/build-async-http-client {} acm)]
```

Comment:
The `false` for caching was completely omitted in this edit because `build-async-http-client` doesn't take a flag for caching (async http doesn't support caching yet).